alchimia/engine.py

```diff
@@ -1,26 +1,58 @@
 from __future__ import absolute_import, division
 
 from sqlalchemy.engine.base import Engine
+from sqlalchemy.exc import StatementError
 
-from twisted.internet.threads import deferToThreadPool
+from twisted._threads import ThreadWorker, AlreadyQuit
```

Comment:
I am taking this comment as an additional prompt to document the intent properly on a Twisted ticket; there was some discussion in comments and on IRC but I didn't see a top-line ticket to move forward on this, so, I've filed 2: one about the API being private specifically, https://twistedmatrix.com/trac/ticket/8846, and a second about @lvh's point about the code being generic https://twistedmatrix.com/trac/ticket/8847.  Both link back to this PR.